pkg/operator/controller/gatewayapi/controller.go

```diff
@@ -2,6 +2,8 @@ package gatewayapi
 
 import (
 	"context"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
```

Comment:
*nit*
We should move this into it's own "group" below `"k8s.io/apimachinery/pkg/types"` I suppose.